### ISSUE TITLE
Don't show emoji selected in commit text

### ISSUE THREAD

**Issue by gollum23** [syuchan1005/GitPrefix#28]
**Describe the bug**
I downloaded the latest zip file from github, I could install but the error about the replace text problem continues

**Screenshots**
![Screenshot_20200702_103322](https://user-images.githubusercontent.com/1058783/86378614-7899b900-bc4f-11ea-945b-faca96b5a118.png)

**Desktop**
- IDE: Pycharm 2020.1
- JRE: Java 11.0.7+10b765.53 amd64
- JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
- Platform: Manjaro Linux KDE
![Screenshot_20200702_103703](https://user-images.githubusercontent.com/1058783/86379073-fb227880-bc4f-11ea-9fd4-9d842103f671.png)

**.gitprefix file**
```text
:wrench:                Update
:bug:                   Bug Fixing
:horse_racing:          Performance
:construction:          WIP
:warning:               Migration
:lock:                  Security Fix
:recycle:               Refactoring
:art:                   Cosmetic
:books:                 Documentation
:no_entry:              Deletion
:white_check_mark:      Addition
:globe_with_meridians:  Translations
:sparkles:              New Feature
:police_car:            Test
:rocket:                Feature
:arrow_double_up:       Upgrade

```

**Comment by KenmuHuang** [comment]
AppCode 2020.1 also have the same issue, can't replace it with selected item. Please fix it, thanks.

**Comment by syuchan1005** [comment]
Fixed in v2.1.9.
Please wait for Jetbrains' approval.
